<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QDialog" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iniciar sesión</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A4161;
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font: 75 30pt &quot;Corbel&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Iniciar sesión</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>431</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>79</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>No. Cliente:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNoCliente">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>196</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>¿Aún no eres cliente?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	color: #2A4161;
	text-decoration: underline;
}

QPushButton:Hover{
	color: #C2C9F2;
}
</string>
         </property>
         <property name="text">
          <string>Registrate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="EtiquetaLogin_Alternativo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> ¿No puedes acceder? </string>
      </property>
     </widget>
     <widget class="QPushButton" name="clickaqui">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Click Aqui</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/park_w.png</normaloff>:/img/img/park_w.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>461</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre(s):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNombre">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido paterno:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtApPat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido materno: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtApMat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Direccion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Teléfono:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha de necimiento:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dteFechaNac">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #A1C1E6;
font: 10pt &quot;Corbel&quot;;
color: black,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRegistarr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>174</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>¿Ya eres cliente?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	color: #2A4161;
	text-decoration: underline;
}

QPushButton:Hover{
	color: #C2C9F2;
}
</string>
         </property>
         <property name="text">
          <string>Inicia sesión</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font: 75 30pt &quot;Corbel&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Registrate</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>441</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inicio de sesion alternativo </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricula: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMatricula">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BotonEntrarAlternativo">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>¿No estás registrado?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>¿Recordaste tu ID ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IniciaSesionAlt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	color: #2A4161;
	text-decoration: underline;
}

QPushButton:Hover{
	color: #C2C9F2;
}
</string>
      </property>
      <property name="text">
       <string>IniciaSesion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RegistrarseAlt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	color: #2A4161;
	text-decoration: underline;
}

QPushButton:Hover{
	color: #C2C9F2;
}
</string>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="iconologinalternativo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>151</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/park_w.png</normaloff>img/park_w.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font: 75 30pt &quot;Corbel&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Inicio de sesion alternativo</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
